EmployeeManagementService/EmployeeManagementService.Domain/Models/Employee.cs

```diff
@@ -32,6 +32,16 @@ public class Employee
         
         public bool? Active { get; set; }
 
+        public string AddressLine1 { get; set; }

```

Comment:
Let's create an Address class in our Domain instead.

Add this new object to Employee and DTO map to an Address object